setup.py

```diff
@@ -45,6 +45,7 @@
         'python-dateutil>=2.6.0,<2.7.0',
         'PyYAML>=5.1',
         'requests>=2.10.0',
+        'tzlocal<3.0',
```

Comment:
This line needs to be removed since it already exists at the bottom of this list.